modules/ROOT/pages/rtm-agent-whitelists.adoc

```diff
@@ -13,7 +13,23 @@ image:logo-server-active.png[link="/runtime-manager/deployment-strategies", titl
 
```

Comment:
INSTRUCTIONS FOR EDIT

a wee PR for your Monday enjoyment. Green only (you've seen this all before):